include/swift/AST/DiagnosticsParse.def

```diff
@@ -925,6 +925,8 @@ ERROR(statement_begins_with_closure,none,
       "top-level statement cannot begin with a closure expression", ())
 ERROR(statement_same_line_without_semi,none,
       "consecutive statements on a line must be separated by ';'", ())
+ERROR(statement_misspell,none,
+    "did you misspell '%0'", (StringRef))
```

Comment:
Some nits:
1) The phrasing here is of a note, but the diagnostic is an error.  Take a look at how we do typo correction diagnostics in libSema.
2) Rhetorical questions need punctuation

Reply:
Agree. Fixed.